dts/bindings/fuel-gauge/sbs,default-sbs-gauge.yaml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+compatible: "sbs,default-sbs-gauge"
+
+include: ["sbs,sbs-gauge-new-api.yaml", "battery_cutoff.yaml"]
+
+description: |
+  Default generic smart battery fuel gauge driver. Includes support for battery cutoff if enabled.
+
+  This compatible is intended to be used with the abstract SBS Gauge compatible because it is
+  actuated by the SBS driver for SBS compliant fuel gauge ICs.
+
+properties:
+  battery_cutoff_support:
+    default: 0
```

Comment:
If you switch this to a bool, then you don't need to set the default value in the yaml schema.